dp-game-webapp/app/game/play/page.tsx

```diff
@@ -30,60 +34,156 @@ export default function Play() {
+                      typical usage. As a reminder, below is your current
+                      configuration.
+                    </p>
+
+                    <div className="mt-6 flex-col items-center justify-between text-lg font-semibold underline underline-offset-auto">
+                      Campaign Stats
+                    </div>
+                    <div className="mb-6 flex-col items-center justify-between">
+                      Number of Impressions: {impressions.toLocaleString()}
+                      <br />
+                      Expected Number of Conversions (Total):{" "}
+                      {totalConversions.toLocaleString()}
+                      <br />
+                      Expected Conversions per 1000 Impressions:{" "}
+                      {conversionPerThousand.toLocaleString()}
+                    </div>
+
+                    <div className="flex justify-between items-center mt-10">
+                      <Link href="/game/validate">
+                        <button className="mt-10 h-12 w-32 bg-red-400 hover:bg-red-600 text-white font-bold py-2 px-4 rounded flex items-center justify-between">
+                          <ArrowLeftCircleIcon className="h-8 w-auto" />
+                          Back
+                        </button>
+                      </Link>
+
+                      <button
+                        className="mt-10 h-12 w-56 bg-sky-400 hover:bg-sky-600 text-white font-bold py-2 px-4 rounded flex items-center justify-between"
+                        onClick={handleStartButtonClick}
+                      >
+                        Start <ArrowRightCircleIcon className="h-8 w-auto" />
+                      </button>
+                    </div>
+                  </>
+                ) : (
+                  <>
+                    <h5 className="mb-2 text-3xl font-bold tracking-tight text-gray-500 dark:text-white">
+                      Round {currentQuestionIndex + 1}
+                    </h5>
+                    <div className="mb-6 flex-col items-center justify-between text-lg font-semibold underline underline-offset-auto">
+                      Hypothetical Results
+                    </div>
+
+                    <p className="mb-2 text-4xl text-grey-700 dark:text-grey-400 justify-end text-center dark:text-white">
+                      {conversions[currentQuestionIndex].toLocaleString()}{" "}
+                      conversions
+                    </p>
+                    <div className="mt-6 flex-col items-center justify-between text-lg font-semibold underline underline-offset-auto">
+                      Campaign Stats
+                    </div>
+                    <div className="mb-6 flex-col items-center justify-between">
+                      Number of Impressions: {impressions.toLocaleString()}
+                      <br />
+                      Expected Number of Conversions (Total):{" "}
```

Comment:
This should be called "average" number of conversions you input. If we say "expected", any conversion value if it falls shorter than that value, no one would increase spend.

Reply:
expected and average have the same meaning, but I prefer expected because it's the expectation based on their input.

let's not assume anything about folks behavior, it generally doesn't matter. we're only interested in the change in behavior given any arbitrary behavior.